Tribler/Core/Modules/process_checker.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from Tribler.Core.SessionConfig import SessionStartupConfig
+
+
+LOCK_FILE_NAME = 'tribler.lock'
```

Comment:
I think I don't really understand how the GUI is checking for another running process. I was thinking that the GUI starts up Tribler and Tribler uses the `ProcessChecker` to see if there already is a running process. If so, this is reported to the GUI and the GUI gives a warning or something like that. Are you thinking of another approach?